### PR TITLE
PE-357: Fix externally filed issue against this repository

### PR DESCRIPTION
**Pull request by jdmedlin1**
## 📝 Description
Here is the filed issue: https://github.com/brightsign/dev-cookbook/issues/22

The solution to the initially reported issue can be seen in the changes in the server/index.js file. Headers have been added to alleviate the reported console errors in the inspector. 

Any additional changes are to clean up preexisting code and add documentation.

### CODE REVIEW

File: templates/cra-template-brightsign-dashboard/template/src/autorun.brs [hunk 1]
```diff
@@ -6,12 +6,13 @@ function main()
 	enableLDWS()
   ' Enable SSH
```
Comment (blshukla) [inline templates/cra-template-brightsign-dashboard/template/src/autorun.brs:7]: Please format/align the comments and method calls correctly.

File: templates/cra-template-brightsign-dashboard/template/src/server/index.js
Comment (blshukla) [inline templates/cra-template-brightsign-dashboard/template/src/server/index.js:1]: Does this not need the `app.use` block added in the `templates/cra-template-brightsign-app/template/src/server/index.js` file above?

File: templates/cra-template-brightsign-app/template/src/autorun.brs [hunk 2]
```diff
@@ -8,13 +8,13 @@ function main()
   ' Enable SSH
   enableSSH()
 
+  ' Initialize roNodeJs with path or correct filename, whether webpack is used or not.
+  node_js = CreateObject("roNodeJs", "sd:/dist/backend.js", {message_port: mp, node_arguments: ["--inspect=0.0.0.0:2999"], arguments: []})
```
Comment (blshukla) [inline templates/cra-template-brightsign-app/template/src/autorun.brs:12]: Is moving this line before creating the HTML widget creation, part of the solution?

**Comment by mingjun7** [comment]
Can you add more details what is the problem and how the issues are resolved by the changes here? It's not very obvious from the code changes.

File: templates/cra-template-brightsign-app/template/src/autorun.brs (hunk 2 above)
Comment (jdmedlin1) [reply templates/cra-template-brightsign-app/template/src/autorun.brs:12]: While it does not directly affect the issue's reported error of impacting the chrome inspect, it enables the html to be seen on the display when in this order. Before, the screen was black.

File: templates/cra-template-brightsign-dashboard/template/src/server/index.js
Comment (jdmedlin1) [reply templates/cra-template-brightsign-dashboard/template/src/server/index.js:1]: lines 7 and 8 in this file are app.use() function calls just like in the other car template index.js file

**Review by blshukla — APPROVED** [review]

**Comment by jdmedlin1** [comment]
> Can you add more details what is the problem and how the issues are resolved by the changes here? It's not very obvious from the code changes.

I have added some context to the description of the PR, but basically the change that seems to have fixed the reported console errors in the chrome inspect is adding headers to the express backend server to allow for defined access.

**Review by mingjun7 — APPROVED** [review]

**Pull request merged**